### PR TITLE
Validate changed pack name

### PR DESCRIPTION
**Pull request by tcarmeli1**
## Related Issues
fixes:https://jira-dc.paloaltonetworks.com/browse/CIAC-9898

## Description
validate pack name didnt change

### CODE REVIEW

File: demisto_sdk/commands/validate/tests/BA_validators_test.py [hunk 1]
```diff
@@ -1201,3 +1206,64 @@ def test_IsEntityNameContainExcludedWordValidator(
...
+        ]
+    )
+    
+@pytest.mark.parametrize(
+    "content_item, expected_id, expected_fix_msg",
+    [
+        (
+            create_metadata_object(paths=["name"], values=["changedName"]),
+            "changedName",
+            "Changing pack name back to changedName.",
```
Comment (BEAdi) [inline demisto_sdk/commands/validate/tests/BA_validators_test.py:1252]:
```suggestion
            "HelloWorld",
            "Changing pack name back to HelloWorld.",
```
I think you tested with the opposite values of what you need.
Maybe in the `fix_message` we can add the name that it changed from (`Changing pack name back from {0} to {1}.`).

**Review by BEAdi — COMMENTED** [review]
Looks great!
Following the failure today in the old BA114, I opened [CIAC-9911](https://jira-dc.paloaltonetworks.com/browse/CIAC-9911) to handle the other checks of the original BA114.

File: demisto_sdk/commands/validate/validators/BA_validators/BA114_is_pack_changed.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+from demisto_sdk.commands.common.constants import PACKS_FOLDER
+from demisto_sdk.commands.common.tools import get_pack_name
+from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    GitStatuses,
+    ValidationResult,
+)
+
+ContentTypes = BaseContent
```
[the remark this answers is not in the document]
Comment (tcarmeli1) [reply demisto_sdk/commands/validate/validators/BA_validators/BA114_is_pack_changed.py:14]: Yes.

**Review by jbabazadeh — APPROVED** [review]
Great work! :)
Please fix the unittests and resolve Yuval's comments

**Pull request merged**